source/_integrations/eufy_security.markdown

```diff
@@ -0,0 +1,123 @@
+
+Password:
+  description: Your Eufy Security account password.
+
+{% endconfiguration_basic %}
+
+{% note %}
+If Eufy's servers require CAPTCHA verification during login, you will be prompted to complete the CAPTCHA challenge in the configuration flow.
+{% endnote %}
+
+## Supported devices
+
+The following Eufy Security device types are supported:
+
+- **Cameras**: Indoor cameras, outdoor cameras, floodlight cameras
+- **Video doorbells**: Battery and wired video doorbells
+
+## Supported functionality
+
+### Camera
+
+The camera platform provides live streaming and snapshot capabilities for your Eufy Security cameras and doorbells.
+
+#### Live streaming
+
+The integration supports live streaming from your cameras. When RTSP credentials are configured on your camera through the Eufy Security app, the integration prefers local RTSP streaming for better performance and reduced latency. If local streaming is not available or not configured, it falls back to cloud streaming.
+
+#### Snapshots
+
+Camera snapshots are supported via FFmpeg. To prevent overwhelming your cameras and the Eufy API, snapshots are throttled to one request per 60 seconds per camera.
+
+## Data updates
+
+The Eufy Security integration communicates with Eufy's cloud API to retrieve camera information and stream URLs. The integration maintains a persistent session to minimize re-authentication requests.
+
+When authentication tokens expire, the integration automatically refreshes them. If Eufy's API returns an alternative endpoint (domain switching), the integration will automatically adapt.
+
+## Known limitations
+
+### Cloud dependency
+
+This integration requires communication with Eufy's cloud servers. If Eufy's servers are unavailable, the integration will not function.
+
+### RTSP local streaming
+
+Local RTSP streaming requires manual configuration of RTSP credentials in the Eufy Security app for each camera. Not all camera models support RTSP streaming.
+
+### Snapshot throttling
+
+Snapshots are limited to one per 60 seconds per camera to prevent excessive API calls and potential rate limiting.
+
+## Troubleshooting
+
+### Authentication issues
+
+If you experience authentication failures:
+
+1. Verify your credentials are correct by logging into the Eufy Security app.
+2. Check if your account requires CAPTCHA verification.
+3. Try removing and re-adding the integration.
+
+### Streaming issues
+
+If live streams are not working:
+
+1. Check that your camera is online in the Eufy Security app.
+2. For local RTSP streaming, verify RTSP credentials are configured in the Eufy Security app.
+3. Ensure your Home Assistant instance can reach either the local camera (for RTSP) or Eufy's cloud servers.
+
+### Reauthentication
+
+If your credentials change or become invalid, Home Assistant will prompt you to reauthenticate. Go to **Settings** > **Devices & Services**, find the Eufy Security integration, and follow the reauthentication prompts.
```

Comment:
```suggestion
If your credentials change or become invalid, Home Assistant will prompt you to reauthenticate. Go to {% my integrations title="**Settings** > **Devices & services**" %}, find the Eufy Security integration, and follow the reauthentication prompts.
```
We have a [My link](https://developers.home-assistant.io/docs/documenting/integration-docs-examples?_highlight=link#my-links) for this.